spacy/default_config.cfg

```diff
@@ -2,13 +2,18 @@
 lang = null
 stop_words = []
 lex_attr_getters = {}
+vocab_data = {}
+get_noun_chunks = null
 pipeline = []
 
 [nlp.tokenizer]
 @tokenizers = "spacy.Tokenizer.v1"
```

Comment:
Why aren't all the tokenizer settings explicit here?

Reply:
I haven't integrated the exceptions/prefixes/suffixes/infixes yet – those are currently still read off the `Language.Defaults` in the tokenizer factory and not passed in from the function registry. So the tokenizer currently only has two settings, `token_match` and `url_match`.

Reply:
But they will be handled the same in the end?

Reply:
Yeah, that's the idea. So languages would write:

```ini
[nlp.tokenizer]
@tokenizers = "spacy.Tokenizer.v1"
token_match = null
url_match = {"@language_data": "spacy.xx.url_match"}
exceptions = {"@language_data": "spacy.en.tokenizer_exceptions"}
# etc.
```